src/containers/CustomRuns/CustomRuns.jsx

```diff
@@ -33,13 +33,13 @@ import {
   Table
 } from '@tektoncd/dashboard-components';
 import {
-  Add16 as Add,
-  Calendar16 as CalendarIcon,
-  TrashCan32 as DeleteIcon,
-  Time16 as TimeIcon,
-  Lightning16 as TriggersIcon,
-  UndefinedFilled20 as UndefinedIcon
-} from '@carbon/icons-react';
+  Add,
+  Calendar as CalendarIcon,
+  TrashCan as DeleteIcon,
```

Comment:
size check on TrashCan

Reply:
same as above, should always have been 16 since the styles constrain it to that size anyway